scripts/linters/pylint_extensions.py

```diff
@@ -701,9 +672,9 @@ def check_typeinfo(self, node, node_doc):
             entries = node_doc._parse_section(  # pylint: disable=protected-access
                 _check_docs_utils.GoogleDocstring.re_param_section)
             for entry in entries:
-                if entry.lstrip().startswith('*args') and not (
-                        entry.lstrip().startswith('*args: list(*)')):
+                if entry.lstrip().startswith('*args: list(*)'):
```

Comment:
Is this still needed?

Reply:
Yup, In many places we are using this format. Like here: https://github.com/oppia/oppia/blob/b511dc1f080f79ac72c274ee2c1f3efcc3231f41/core/controllers/acl_decorators.py#L85

But once this `*args: list(*)` format is removed from the codebase. We can remove this test.

Reply:
ah, okay